### DISCUSSION TITLE
Debian 12, xtermjs comes up in page but will not take any input....

### DISCUSSION THREAD

**Discussion by N6GKJ** [sourcelair/xterm.js#5004] [Q&A]
Xtermjs seems to come up in browser, but nothing happens... tells me 'Hello from xterm.js >'
Wont accept any input, no cursor

Installed with npm install @xterm/xterm
is there more to install?

**Comment by N6GKJ** [comment]
My goal is to run 'asterisk -rvvvv' and interact with the application, Will this do that?

**Comment by jerch** [comment]
> My goal is to run 'asterisk -rvvvv' and interact with the application, Will this do that?

I dont know anything about asterisk, but if -rvvvv gives you an interactive terminal interface, then chances are high that xterm.js can properly render its output.

> Xtermjs seems to come up in browser, but nothing happens... tells me 'Hello from xterm.js >'
   ...
   is there more to install?

Ofc there is, xterm.js is just the terminal emulator providing keyboard input and screen output. You will also need a connection (e.g. via websocket) to a PTY running some process (typically a shell).
